2.7.0/done/result_quantization.md

```diff
@@ -27,42 +27,93 @@ The categories below are as follows:
 
 ## quantization
 ### bc breaking
-- [BC-Breaking]Remove capture_pre_autograd_graph references in quantization ([#139505](https://github.com/pytorch/pytorch/pull/139505))
+- Remove capture_pre_autograd_graph references in quantization ([#139505](https://github.com/pytorch/pytorch/pull/139505))
+
+```python
+# pytorch 2.6
+from torch._export import capture_pre_autograd_graph
+from torch.ao.quantization.quantize_pt2e import prepare_pt2e
+from torch.ao.quantization.quantizer.xnnpack_quantizer import (
+    XNNPACKQuantizer,
+    get_symmetric_quantization_config,
+)
+quantizer = XNNPACKQuantizer().set_global(
+    get_symmetric_quantization_config()
+)
+m = capture_pre_autograd_graph(m, *example_inputs)
+m = prepare_pt2e(m, quantizer)
+
+# pytorch 2.7
+from torch.export import export_for_training
+from torch.ao.quantization.quantize_pt2e import prepare_pt2e
+# please get xnnpack quantizer from executorch (https://github.com/pytorch/executorch/)
+from executorch.backends.xnnpack.quantizer.xnnpack_quantizer import (
+    XNNPACKQuantizer,
+    get_symmetric_quantization_config,
+)
+quantizer = XNNPACKQuantizer().set_global(
+    get_symmetric_quantization_config()
+)
+m = export_for_training(m, *example_inputs)
+m = prepare_pt2e(m, quantizer)
+```
+
 ### deprecation
+- Add deprecation warning to XNNPACKQuantizer ([#144940](https://github.com/pytorch/pytorch/pull/144940))
+
+```python
+# pytorch 2.6
+from torch._export import capture_pre_autograd_graph
+from torch.ao.quantization.quantize_pt2e import prepare_pt2e
+from torch.ao.quantization.quantizer.xnnpack_quantizer import (
+    XNNPACKQuantizer,
+    get_symmetric_quantization_config,
+)
+quantizer = XNNPACKQuantizer().set_global(
+    get_symmetric_quantization_config()
+)
+m = capture_pre_autograd_graph(m, *example_inputs)
+m = prepare_pt2e(m, quantizer)
+
+# pytorch 2.7
+# we also updated the export call
+from torch.export import export_for_training
+from torch.ao.quantization.quantize_pt2e import prepare_pt2e
+# please get xnnpack quantizer from executorch (https://github.com/pytorch/executorch/)
+from executorch.backends.xnnpack.quantizer.xnnpack_quantizer import (
+    XNNPACKQuantizer,
+    get_symmetric_quantization_config,
+)
+quantizer = XNNPACKQuantizer().set_global(
+    get_symmetric_quantization_config()
+)
+m = export_for_training(m, *example_inputs)
+m = prepare_pt2e(m, quantizer)
+```
+
+
 ### new features
+- debug handler maintain through decomposition ([#141612](https://github.com/pytorch/pytorch/pull/141612))
+- Add support for list, tuple and dict in numeric debugger ([#143882](https://github.com/pytorch/pytorch/pull/143882))
+
 ### improvements
+- Add an option keep_original_weights in _lower_to_native_backend ([#141049](https://github.com/pytorch/pytorch/pull/141049))
```

Comment:
```suggestion
- Add an option `keep_original_weights` in `_lower_to_native_backend` (#141049)
```

Remember to backtick code + shorten the PR link for this entry + every other entry. A quick sed command to do the PR link thing whenever you're ready: sed -i 's/\[#\([0-9]\+\)\](https:\/\/github\.com\/pytorch\/pytorch\/pull\/[0-9]\+)/#\1/g' <inputfile.md>

Reply:
I used to do it, but found that it actually does not really affect how it is displayed

Reply:
Right, but we want the raw source markdown to be shorter since it displays the same thing

Reply:
is this for fitting more characters into release note? can you do it in the end if it's too long?

I mean I can do it, it's just asking everyone to do it seems to be less efficient than doing it once when needed.

Reply:
Hmmmm that's a good point tbh, I will update the instructions for next time about that.